.github/CONTRIBUTING.md

```diff
@@ -1,147 +1,89 @@
-# Contributing
+# Welcome to the Frontend Contributing Guide!
 
-HospitalRun is a community project. We invite your participation through
-financial contributions, issues, and pull requests!
+Before contributing, please read the [code of conduct](https://github.com/HospitalRun/hospitalrun/blob/master/.github/CODE_OF_CONDUCT.md).
 
-Contributions are always welcome. Before contributing please read the [code of conduct](https://github.com/HospitalRun/hospitalrun/blob/master/.github/CODE_OF_CONDUCT.md) and
-[search the issue tracker](https://github.com/HospitalRun/hospitalrun-frontend/issues); your issue
-may have already been discussed or fixed in `master`. If you're new to the project,
-maybe you'd like to open a pull request to address one of [good-first-issue](https://github.com/HospitalRun/hospitalrun-frontend/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
+## 1. Check out the [HospitalRun General Contributing Guide](https://github.com/HospitalRun/hospitalrun/blob/master/.github/CONTRIBUTING.md) for how to:
+   1. Browse Issues
+   2. Get Assigned an Issue
+   3. Fork the Repository and Create a Branch
+   4. Commit Changes
+   5. Submit a Pull Request
 
-Make sure to checkout the [HospitalRun Contributing Guide](https://github.com/HospitalRun/hospitalrun/blob/master/.github/CONTRIBUTING.md)
+## 2. After creating a local branch (step 3 above), follow these steps to configure CouchDB
```

Comment:
I think we should change to `After creating a local fork...`

Reply:
✅